src/music21/base.ts

```diff
@@ -484,6 +484,20 @@ export class Music21Object extends prebase.ProtoM21Object {
             }
         }
     }
+
+    repeatAppend(object, item, numberOfTimes) {
```

Comment:
Remember that in Javascript there is no need for the `object` here to substitute for Python's `self` argument.  Instead `self` becomes `this`.  So in the `object.append` part it will just be `this`.